### PR TITLE
Additional note.

### PR DESCRIPTION
**Pull request by jasonluiscruz**
Added the information that the NDI is activated per session only. On next login user must activate it before using NDI.
Product Group information based on ICM https://portal.microsofticm.com/imp/v3/incidents/details/230031375/home

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/use-NDI-in-meetings.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ The following image shows the banner message that a user sees in a Teams meeting
 
 The banner has a link to the [Microsoft privacy policy](https://aka.ms/teamsprivacy).
 
+[!NOTE] NDI is activated per session only. On next login user must activate it before using NDI.
```
Comment (JohanFreelancer9) [inline Teams/use-NDI-in-meetings.md:51]:
```suggestion
> [!NOTE]
> NDI® is activated per session only. On the next login, the user must activate it before using NDI®.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@jasonluiscruz - please commit my suggested changes. Thanks.

**Comment by jasonluiscruz** [comment]
> @jasonluiscruz - please commit my suggested changes. Thanks.

**Comment by JohanFreelancer9** [comment]
@jasonluiscruz - you need to reopen this PR so that it can be merged, unless there is a reason that you closed it? please advise.

**Comment by jasonluiscruz** [comment]
> @jasonluiscruz - you need to reopen this PR so that it can be merged, unless there is a reason that you closed it? Please advise.

I've reopened it, not sure how the PR got closed. My bad.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**